extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -545,3 +553,22 @@ export async function promptAndUpdateConnectionStringSetting(projectUri: vscode.
 	}
 	return connectionStringSettingName;
 }
+
+export async function promptConnectionStringPassword(connectionString: string, connectionInfo?: IConnectionInfo): Promise<string> {
+	// prompt user for entering in password or not
+	// Ask user to enter password if auth type is sql login and password is not saved
+	if (connectionInfo?.authenticationType === 'SqlLogin') {
+		let userPassword = await vscode.window.showInputBox({
+			prompt: constants.enterPasswordPrompt,
+			placeHolder: constants.enterPasswordManually,
+			ignoreFocusOut: true,
+			password: true,
+			validateInput: input => input ? undefined : constants.valueMustNotBeEmpty
+		});
+		if (userPassword) {
+			// if user enters password replace password placeholder with user entered password
+			connectionString = connectionString.replace(constants.passwordPlaceholder, userPassword);
```

Comment:
What if the password doesn't have a placeholder at all - or it's a different type of placeholder? We should be handling those scenarios too best we can.